test/gc/binding.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+try {
+  module.exports = require('./build/Release/binding').ongc;
+} catch (e) {
+  module.exports = require('./build/Debug/binding').ongc;
```

Comment:
Try to load the version that matched the mode (release/debug) of the node process first and fallback to the other?